insights/parsers/proc_limits.py

```diff
@@ -62,9 +76,16 @@ def __iter__(self):
 
     def parse_content(self, content):
         self.data = []
-        self.max_processes = ''
         if 'No such file or directory' not in content[0]:
             self.data = parse_fixed_table(content, header_substitute=HEADER_SUBSTITUTE)
```

Comment:
@xiangce  should you throw a `ParseException` here to indicate that no data is available?

Reply:
@bfahr - Thanks.
I think it's OK to throw a `ParseException`.
However, for the hosts which are not running `httpd`, the pre_command `httpd_pid` will return empty. I'm not sure in that case if the CommandSpec will be executed by our client. (In that case the CommandSpec will be `cat /proc//limits` which always results in "cat: /proc//limits: No such file or directory".) If the CommandSpec continues to be executed, there will be a lot of `ParseException` raised. Does this make sense?

Reply:
If there will be no data in the output then either an exception should be thrown to let the core know that data should not be stored.  We are trying to avoid the situation where rules have to get the parser data and then have to check for `None`.

Reply:
Got it. Will do it.